docs/src/analysis/sensitivity.md

```diff
@@ -383,3 +385,185 @@ end
+
+  a) Pearson Correlation Coefficient:
+
+$$ r = \frac{\sum_{i=1}^{n} (x_i - \overline{x})(y_i - \overline{y})}
+{\sqrt{\sum_{i=1}^{n} (x_i - \overline{x})^2(y_i - \overline{y})^2}} $$
+
+  b) Standard Regression Coefficient (SRC):
+
+$$ SRC_j = \beta_{j} \sqrt{\frac{Var(X_j)}{Var(Y)}} $$
+
+  where $$ \beta_j $$ is the linear regression coefficient associated to $X_j$.
+
+  c) Partial Correlation Coefficient (PCC):
+
+$$ PCC_j = \rho(X_j - \hat{X_{-j}},Y_j - \hat{Y_{-j}}) $$
+
+  where $$ \hat{X_{-j}} $$􏰈 is the prediction of the linear model, expressing $$ X_{j} $$ 
+  with respect to the other inputs and $$ \hat{Y􏰈_{-j}} $$ is the prediction of the 
+  linear model where $$ X_j $$ is absent. PCC measures the sensitivity of $$ Y $$ to 
+  $$ X_j $$ when the effects of the other inputs have been canceled.
+
+`regre_sensitivity = regression_sensitivity(f,param_range,param_fixed,n;coeffs=:rank)`
+
+`regre_sensitivity = regression_sensitivity(prob::DiffEqBase.DEProblem,alg,t,param_range,param_fixed,n;coeffs=:rank)`
+
+Again, `f` and `param_range` are the same as above. An array of the true parameter values 
+that lie within the `param_range` bounds are passed through the `param_fixed` argument. 
+`n` determines the number of simulations of the model run to generate the data points 
+of the solution and parameter values and the `coeffs` kwarg lets you decide the
+coefficients you want.
+
+### GSA example 
+
+Let's create the ODE problem to run our GSA on.
+
+```julia
+f = @ode_def_nohes LotkaVolterraTest begin
+    dx = a*x - b*x*y
+    dy = -3*y + x*y
+end a b 
+u0 = [1.0;1.0]
+tspan = (0.0,10.0)
+p = [1.5,1.0]
+prob = ODEProblem(f,u0,tspan,p)
+t = collect(range(0, stop=10, length=200))
+```
+For Morris Method
+
+```julia
+m = DiffEqSensitivity.morris_sensitivity(prob,Tsit5(),t,[[1,5],[0.5,5]],[10,10],len_trajectory=1500,total_num_trajectory=1000,num_trajectory=150)
+```
+Let's get the means and variances from the `MorrisSensitivity` struct.
+
+```julia
+m.means
+
+Out[9]: 2-element Array{Array{Float64,2},1}:
+ [0.0 0.0513678 … 7.91336 7.93783; 0.0 0.00115769 … 3.66156 3.67284]
+ [0.0 0.0488899 … 2.50728 2.359; 0.0 0.00112006 … 2.23431 2.44946]  
+
+m.variances
+
+Out[10]: 2-element Array{Array{Float64,2},1}:
+ [0.0 1.94672e-5 … 26.4223 24.8513; 0.0 4.81347e-9 … 37.4061 30.3068]
+ [0.0 1.77615e-5 … 17.9555 14.9231; 0.0 4.47931e-9 … 48.074 51.9312] 
+```
+This gives the means of the effects and it's variances over the entire timespan and thus we get 200-length 
+arrays for each paramter and dependent variable pair.
+
+We can plot the trajectory of the sensitivity with the standard deviation bars.
+```julia
+# For the first parameter (a)
+stdv1 = sqrt.(m.variances[1])
+p = plot(m.means[1]', yerror=stdv1)
+```
+![morrisparameter1](../assets/morris1.png)
+
+```julia
+# For the second parameter (b)
+stdv2 = sqrt.(m.variances[2])
+p = plot(m.means[2]', yerror=stdv2)
+```
+![morrisparameter2](../assets/morris2.png)
```

Comment:
Now show that you can get the sensitivity only at prescribed time points by using `saveat`.